libass/ass.h

```diff
@@ -734,6 +734,24 @@ void ass_process_chunk(ASS_Track *track, const char *data, int size,
  */
 void ass_set_check_readorder(ASS_Track *track, int check_readorder);
 
+/**
+ * \brief Prune events that preceed deadline.
+ * \param track track
+ * \param deadline cut-off timestamp in milliseconds.
+*/
+void ass_prune_events(ASS_Track *track, long long deadline);
+
+/**
+ * \brief Configure automatic pruning of events.
+ * \param track track
+ * \param delay delay from "now" (ass_render_frame) in milliseconds.
+ * After every succesful render, events whose undisplay timestamp
+ * predate "now - delay" will be deleted.
+ * Negative delays disable pruning. A delay of 0 prunes aggressively.
```

Comment:
*(for some reason this disappeared from the initial approving review)*

>  Negative delays disable pruning.

To avoid confusion abouth whether this affects manual `ass_prune_events` calls, i think *“Negative delays disable **automatic pruning.”* would be better